src/test/groovy/de/dkfz/roddy/batcheuphoria/jobs/BEJobIDSpec.groovy

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2017 eilslabs.
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ */
+
+package de.dkfz.roddy.batcheuphoria.jobs
+
+import de.dkfz.roddy.execution.jobs.BEJobID
+import spock.lang.Specification
+
+class BEJobIDSpec  extends Specification {
```

Comment:
?? What does it do?

Reply:
Test the equals and hashcode functions coming from the annotation?